path

```diff
@@ -0,0 +1,39 @@
+--- docs/hsi-tests.d/org.fwupd.hsi.Amd.SmmLocked.json
```

Comment:
I'm not sure you meant to commit this file.

Reply:
Whoops thanks!